### ISSUE TITLE
Whitespace and indentation a little off for inline help

### ISSUE THREAD

**Issue by arr2036** [alandekok/recli#9]
```
nm-subscriber> show subscriber ?
	Show the status of a subscriber using various identifying attributes.
	id: Show the subscriber with the specified id
	ip: Show the subscriber with the specified ip (may be satic or dynamic)
	mdn: Show the subscriber with the specified mdn
	username: Show the subscriber with the specified username
```

Would be good to have a newline after the description of the current node.

Indenting the help text would help readability a lot

```
nm-subscriber> show subscriber ?
	Show the status of a subscriber using various identifying attributes.

	id        Show the subscriber with the specified id
	ip        Show the subscriber with the specified ip (may be satic or dynamic)
	mdn       Show the subscriber with the specified mdn
	username  Show the subscriber with the specified username
```

```
# show

    Show details of subscribers or sessions.

## show subscriber

    Show the status of a subscriber using various identifying attributes.

### show subscriber id

    Show subscriber by id.

### show subscriber ip

    Show subscriber by ip (may be satic or dynamic).

### show subscriber username

    Show subscriber by username.

### show subscriber mdn

    Show subscriber by msisdn.

## show session

    Show sessions by apn or subscriber.

Displays either the count of a particular type of session, or the full details
of all sessions matching the filter. 

Sessions may be filtered by apn or subscriber, and again by whether the session
is currently active.

### show session apn

    Show sessions belonging to all subscribers on a particular apn.

When displaying session counts, the special apn name ``all`` may be used to
display a summary of sessions across all apns.  You cannot display all
historical or active sessions for all apns.

#### show session apn all count

   Show session counts for all apns.

#### show session apn all active count

   Show active session counts for all apns.

#### show session apn STRING

   Show all sessions for this apn.

#### show session apn STRING active

   Show active sessions for this apn.

#### show session apn STRING count

   Show a count of all sessions for this apn.

#### show session apn string count active

   Show a count of all active sessions for this apn.

### show session subscriber

    Show all sessions for a subscriber.

#### show session subscriber mdn

    Show a count of all sessions by subscriber msisdn.

#### show session subscriber mdn active

    Show active sessions for subscriber.
```

**Comment by alandekok** [comment]
better now

**Comment by alandekok** [comment]
fixed by commit 17fc8d86
